website/server/models/group.js

```diff
@@ -508,7 +508,7 @@ schema.statics.collectQuest = async function collectQuest (user, progress) {
 };
 
 schema.statics.bossQuest = async function bossQuest (user, progress) {
-  let group = await this.getGroup({user, groupId: 'party'});
+  let group = await this.getGroup({user, groupId: 'party'}); // TODO Is the new form of double cron sneaking in here? getGroup() runs cron and bossQuest() runs within cron. Example of bug: https://github.com/HabitRPG/habitrpg/issues/2805#issuecomment-222314260
```

Comment:
this is something different. `this.getGroup` is a function of the Group model while api.getGroup in controllers/api-v3/groups.js is a route, the two things are different